libkbfs/journal_md_ops.go

```diff
@@ -222,8 +222,8 @@ func (j journalMDOps) getRangeFromJournal(
 	return irmds, nil
 }
 
-func (j journalMDOps) GetForHandle(
-	ctx context.Context, handle *TlfHandle, mStatus MergeStatus) (
+func (j journalMDOps) GetForHandle(ctx context.Context, handle *TlfHandle,
+	mStatus MergeStatus, lockBeforeLock *keybase1.LockID) (
```

Comment:
`lockBeforeLock` -> `lockBeforeGet` (here and everywhere below).

Reply:
OOPS thanks!